Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+1. **Submit a Pull Request**: Submitting a Pull Request is the technical term for submitting the suggested edits to be reviewed, approved by someone else and updated on the site. It is so named because the reviewer will be 'pulling' your code into the vmware repository to merge it. 
+1. **Delete**: When the change is approved, delete your copy of the practices site.
+
+Repeat steps when you want to suggest new edits to the site.
+
+# Step by Step Instructions 
+
+## 1. Prepare
+1. Type up the changes you want to make for the Practices site. Any word editor will do for this, Microsoft Word, Google Docs, Notes on Mac. For your first suggestion, try adding something small, such as a few sentences. This will make it easier for the suggestion to be approved and merged.
+1. Create a Github account, if you don’t have one. Login to your Github account, if you do.
+1. If you’re doing this for the first time, set aside around 35 minutes to go through these steps.
+
+## 2. Fork (Create a copy of Practices site)
+1. Go to the [VMware practices repository on Github](https://github.com/vmware-tanzu/tanzu-dev-portal/tree/main/content/practices) - this is where all the code for the site sits. It is also open to the public for contribution. 
+1.  Click on fork to make a copy of the code to your own github account.
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork2.png" />
+1. Check the title of the repository to make sure that you are editing on your own copy of the code. It should show your github account name/tanzu-dev-portal, as you can see below:
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork3.png" />
+
+
+1. Click on Content
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork4.png" />
+1. Click on Practices. You’ll see a list of all the practices currently on the site.
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork5.png" />
+1. Click on the practice you want to add to. For this example, I clicked on design-studio.
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork6.png" />
+1. Click on index.md. This shows you what the page looks like in plain text. 
+
+1. Click on the pencil edit icon. This shows you what the page looks like in markdown language. Don’t worry, markdown is a simplified HTML/CSS script.   
```

Comment:
```suggestion
1. Click on the pencil edit icon. This shows you what the page looks like in markdown language. Don’t worry, markdown is a simplified language for formatted text.   
```

Reply:
Maybe we also consider adding a link to a markdown guide that includes the basic formatting options, how to include images, etc.  I think GitHub has a good one somewhere.

Reply:
Guide here: https://guides.github.com/features/mastering-markdown/